### PR TITLE
add missing keys

### PR DESCRIPTION
**Pull request by jeffersondarcy**
I have added keys for components inside map to avoid warnings

### CODE REVIEW

File: src/modals/DeactivateAccountModal.tsx [hunk 1]
```diff
@@ -104,7 +104,9 @@ const DeactivateAccountModal: React.FC<Props> = ({ isOpen, onCloseModal }) => {
                         <Radio.Group name="reasons" onChange={setReason}>
                             <VStack space={space['0.5']}>
                                 {reasons.map((_, index: number) => (
-                                    <Radio value={`${index + 1}`}>{t(`profile.Deactivate.${userType}.${index + 1}`)}</Radio>
+                                    <Radio key={index} value={`${index + 1}`}>
```
Comment (jeffersondarcy) [inline src/modals/DeactivateAccountModal.tsx:107]: had to use index here, feel free to suggest something better

File: src/modals/DissolveMatchModal.tsx [hunk 2]
```diff
@@ -34,7 +34,9 @@ const DissolveMatchModal: React.FC<DissolveModalProps> = ({ showDissolveModal, o
                             <Radio.Group name="dissolve-reason" value={reason} onChange={setReason}>
                                 <VStack space={space['1']}>
                                     {reasons.map((_: number, index: number) => (
-                                        <Radio value={`${index + 1}`}>{t(`matching.dissolveReasons.${userType}.${index + 1}`)}</Radio>
+                                        <Radio key={index} value={`${index + 1}`}>
```
Comment (jeffersondarcy) [inline src/modals/DissolveMatchModal.tsx:37]: didn't find anything better than index here

File: src/pages/pupil/Dashboard.tsx [hunk 3]
```diff
@@ -319,7 +319,7 @@ const Dashboard: React.FC<Props> = () => {
                                         if (!course) return <></>;
 
                                         return (
-                                            <Column minWidth="230px" maxWidth="300px" flex={1} h="100%">
+                                            <Column minWidth="230px" maxWidth="300px" flex={1} h="100%" key={`${course.course.description}+${lecture.start}`}>
```
Comment (jeffersondarcy) [inline src/pages/pupil/Dashboard.tsx:322]: id was not working here @Jonasdoubleyou I don't think the line `navigate(`/single-course/${course.id}`);` below works

Comment (Jonasdoubleyou) [reply src/pages/pupil/Dashboard.tsx:322]: Well, there can be multiple appointments (aka lectures) for the same subcourse. The page is for a subcourse though, so multiple appointments will open the same subcourse page.

**Review by Jonasdoubleyou — APPROVED** [review]

File: src/pages/pupil/Dashboard.tsx (hunk 3 above)
Comment (jeffersondarcy) [reply src/pages/pupil/Dashboard.tsx:322]: I have tried to use course.id for index, but it was apparently undefined. Maybe it works in some other branch

**Pull request merged**